app/views/hostgroups/_form.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= javascript 'host_edit', 'host_edit_interfaces' %>
 <%= form_for @hostgroup, :html => {:class => 'hostresource-form form-horizontal well', :data => {:id => @hostgroup.try(:id)}} do |f| %>
   <%= base_errors_for @hostgroup %>
+  <%= react_component('ResourceErrors', host_errors(@hostgroup).to_json) %>
```

Comment:
Can't we have ResourceErrors in separater PR?
It doesn't sound strictly relevant, the slot could be simple merge without it.